feature-libs/cart/base/components/cart-proceed-to-checkout/cart-proceed-to-checkout.component.html

```diff
@@ -0,0 +1,12 @@
+<cx-progress-button
+  (clickEvent)="disableButtonWhileNavigation()"
+  [class]="'btn btn-primary btn-block'"
+  [disabled]="cartValidationInProgress"
+  [loading]="cartValidationInProgress"
+  [routerLink]="{ cxRoute: 'checkout' } | cxUrl"
+>
+  {{
+    (!cartValidationInProgress ? 'cartDetails.proceedToCheckout' : '')
```

Comment:
cart-lib fixed this, where they added text, but that's an issue since text wrapped due to the length of it. https://github.com/SAP/spartacus/issues/14952

Currently causing the same issue as it was pre-cart-lib merge. To discuss with Patrick and Wei on a fix.

Reply:
Basically, Patrick and Wei added text because when passing an empty string, there's an error in devtools

Reply:
I saw it, there is already Pr with Fix so I have updated this part in my PR also.